docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  mqtt_master:
+    image: chrisns/docker-vernemq:addwaitforit
```

Comment:
I guess this one should point to `erlio/docker-vernemq` in the end, right? 
Same on line 20